lib/screens/customer_product_detail_page.dart

```diff
@@ -84,18 +84,44 @@ class _CustomerProductDetailPageState extends State<CustomerProductDetailPage> {
       ),
     );
     bodyWidgets.add(SizedBox(height: 40));
+    bodyWidgets.add(Container(
+      decoration: BoxDecoration(
+        border: Border(top: BorderSide(color: kPaletteBorderColor)),
+      ),
+      child: Padding(
+        padding: const EdgeInsets.only(top: 8),
+        child: product.stock.stockType == StockType.quantityOnly
+            ? Container()
+            : Text(
+                _getSelectStockDisplayText(product.stock.stockType),
+                style: TextStyle(color: paletteBlackColor),
+              ),
+      ),
+    ));
+    bodyWidgets.add(SizedBox(height: 40));
     bodyWidgets.add(_buildStockSelector(product));
     bodyWidgets.add(SizedBox(height: 150));
 
     return bodyWidgets;
   }
 
+  String _getSelectStockDisplayText(StockType productStockType) {
+    if (productStockType == StockType.colorOnly)
+      return "下記より色をお選びください";
+    else if (productStockType == StockType.sizeAndColor)
+      return "下記よりサイズと色をお選びください";
+    else if (productStockType == StockType.sizeOnly)
+      return "下記よりサイズをお選びください";
+    else
+      return null;
```

Comment:
@dragonfly91 How do you differentiate `switch` and `if/else`? 🙂

Reply:
Hi @kumiko-haraguchi I'm not sure if I understand your question correctly. Do you mean, when to use switch and when to use if/else? If this is your question, I think both are interchangeable, but I felt the if-else implementation to be more compact in this instance.

Reply:
@dragonfly91 I've seen you using `switch` so far and thought you have a reason to chose it. I didn't see any significant difference in this `_getSelectStockDisplayText` and other `get` methods with switch statement so I wondered how you decide when to use `switch` and `if/else`. That'll help me to follow when I encounter the situation in the future. :)

Reply:
I think we currently don't have any strong opinion towards using switch vs if/else, so feel free to use whichever makes the most sense to you! In this case, I meant 'compact' in the sense of lines of code :)

Reply:
@dragonfly91 I see, ok, got it!!! :D

Reply:
Awesome! If you're satisfied with this PR, can you please resolve this comment and approve the PR? @kumiko-haraguchi